firestore.rules

```diff
@@ -1,8 +1,81 @@
 rules_version = '2';
 service cloud.firestore {
+	function notUpdating(field) {
+   return !(field in request.resource.data)
+    || resource.data[field].matches(request.resource.data[field])
+  }
+  
+  function checkUserFieldsValid(){
```

Comment:
Not super important but the indentation of line # 3 seems off and the spacing between the closing parenthesis and the opening brace is inconsistent

Reply:
Thanks, somehow it was like a half space? :)